imi/src/main/java/org/motechproject/nms/imi/service/impl/SecureCopy.java

```diff
@@ -0,0 +1,86 @@
+package org.motechproject.nms.imi.service.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public final class SecureCopy {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SecureCopy.class);
+
+
+    private SecureCopy() { }
+
+
+    private static String getStderr(Process p) {
+        StringBuilder sb = new StringBuilder();
+        BufferedReader b = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+        String line;
+
+        try {
+            while ((line = b.readLine()) != null) {
+                sb.append(line);
+                sb.append('\n');
+            }
+
+            b.close();
+        } catch (IOException e) {
+            LOGGER.error("Error trying to retrieve scp error message: {}", e.getMessage());
+        }
+        return sb.toString();
+    }
+
+
+    public static void copyFrom(String user, String host, String identityFile, String remoteSource,
+                                String localDestination) throws SecureCopyException {
+
+        Runtime r = Runtime.getRuntime();
+        try {
+            String identity;
+            if (identityFile == null || identityFile.isEmpty()) {
+                identity = "";
+            } else {
+                identity = String.format("-i %s ", identityFile);
+            }
+            String command = String.format("scp %s%s@%s:%s %s", identity, user, host, remoteSource, localDestination);
```

Comment:
The space is on line 47.
Agree on the full path / config value (defaulting to /usr/bin/scp).